Relativity.Transfer.SDK.md

```diff
@@ -4,6 +4,33 @@
 
 The Relativity Transfer .NET SDK package.
 
+## v2.0.0
+
+### Release Notes
+
+#### Breaking changes:
+- `AzureTransferFullPathClientBuilder` and `AzureTransferJobClientBuilder` now require a workspace context, which can be specified by the workspace's ID or using Staging Explorer privileges.
+- Cancelled directory transfers will no longer throw `OperationCancelledException`. Instead, a result with a "Cancelled" state will be returned.
+- If the client's implementation of `IRelativityAuthenticationProvider` throws an exception, the SDK will now throw a `FailedToAcquireCredentialsException`.
+
+#### New Features:
+- Introduced `OvewritePolicy` in `OptionsBase` for implementing policies to handle file skipping and overwriting at the destination.
+- `AzureTransferJobClient` now supports uploading a list of files to the fileshare.
+- Added the ability to specify the location of journal files in transfer options.
+
+#### Improvements:
```

Comment:
I would suggest to combine improvements and new features into one section. For example journal removal is feature for me ;)

Reply:
Done